### PR TITLE
Localize home page

### PR DESCRIPTION
**Pull request by alicoding**
@alanmoo R?

### CODE REVIEW

**Comment by alicoding** [comment]
ping @alanmoo R?

File: pages/home/ThankYouModal.jsx [hunk 1]
```diff
@@ -1,18 +1,23 @@
...
+  var twitterShare = "https://twitter.com/home?status=" + encodeURIComponent(twitterShareMessage);
     return (
-      <Modal modalTitle="Thanks for your interest!" className="modal-signup" hideModal={this.props.hideModal}>
-        <p>We appreciate your commitment to keeping the web open, accessible and ours.</p>
-        <p><strong>Share and tell your friends</strong></p>
+      <Modal modalTitle={this.context.intl.formatMessage({id: 'thanks_for_your_interest'})} className="modal-signup" hideModal={this.props.hideModal}>
+        <p><FormattedMessage id='thank_you_modal_message' /></p>
+        <p><strong><FormattedMessage id='thank_you_modal_share_message' /></strong></p>
         <div className="social-share">
-          <a href={facebookShare} className="facebook"><i className="fa fa-facebook"></i>Facebook</a>
```
Comment (alanmoo) [inline pages/home/ThankYouModal.jsx]: I'm not entirely sure brands need to be localized, but I guess it doesn't hurt?

Comment (alicoding) [inline pages/home/ThankYouModal.jsx]: There was a history of disagreement between localizers in different community about this matter, so I want to leave them to make the decision here...

Comment (alanmoo) [inline pages/home/ThankYouModal.jsx]: Good to know.

**Comment by alanmoo** [comment]
LGTM

**Pull request merged**